### ISSUE TITLE
what math form of the loss function ?

### ISSUE THREAD

**Issue by zilesazoyi2** [djangogo/pose-tensorflow-ANNOT#2]
thanks you a lot ,you code and annotations  really save me !

could please give a exact math form of the total loss function? I need it desirous !
thank you again 
打扰您了Ｏ（∩＿∩）Ｏ您的注释对我帮助很大。感谢您的无私精神。　

原先的论文deepercut我没有找到具体的损失函数定义，deepcut中定义的目标函数好像和神经网络没有关系？　
pose-tensorflow最初的版本也没有找到resnet的中损失函数的定义，
我看您的注释猜测：
ｌｏｓｓ是整个ＭＰＩＩ数据库上估计关节点坐标与预测关节点坐标距离的平方和　。
同时没有看懂hearmap是如何在resnet中是如何得到的。
可以麻烦您　给出 total loss function的数学表达式吗？

我的邮箱是zilesazoyi2@163.com
